From 53a8b55839d1cd4be6dc25f11aeb93e8348cc270 Mon Sep 17 00:00:00 2001
From: Nicola Guerrera <guerrera.nicola@gmail.com>
Date: Thu, 30 Oct 2025 17:52:43 +0100
Subject: [PATCH 75/77] dts: qcom: sm8150: xiaomi-nabu: remove wakeup-source
 from lid gpio

This fixes the tablet waking up when the lid is closed
---
 arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts b/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
index dfebc7f3e70c..b7845863eaa1 100644
--- a/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
+++ b/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
@@ -198,7 +198,7 @@ event-hall-sensor {
 			linux,code = <SW_LID>;
 			debounce-interval = <15>;
 			linux,can-disable;
-			wakeup-source;
+			// wakeup-source;
 		};
 	};
 };
-- 
2.51.1

